### PR TITLE
Upgrade dotnet8

### PR DESCRIPTION
**Pull request by jvandaal**
feat: move to dotnet 8.0.2

BREAKING CHANGE: move to dotnet 8.0.2

### CODE REVIEW

File: src/RoadRegistry.RoadNetwork.Schema/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// using System.Reflection;
```
Comment (ArneD) [inline src/RoadRegistry.RoadNetwork.Schema/Properties/AssemblyInfo.cs:1]: TBD what to do

File: test/RoadRegistry.BackOffice.Api.Tests/RoadRegistry.BackOffice.Api.Tests.csproj [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
```
Comment (ArneD) [inline test/RoadRegistry.BackOffice.Api.Tests/RoadRegistry.BackOffice.Api.Tests.csproj:4]: We don't use the Test.props here which also includes targetframework, but i'm not sure how needed this is.

File: src/Be.Vlaanderen.Basisregisters.IdentityServer/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,4 @@
 FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```
Comment (emalfroy) [inline src/Be.Vlaanderen.Basisregisters.IdentityServer/Dockerfile:1]: net8?

**Review by ArneD — DISMISSED** [review]

**Comment by rikdepeuter** [comment]
Current version of Amazon.Lambda.Core breaks SQS integration through localstack

**Pull request merged**